gradle-plugin/src/main/kotlin/arrow/meta/plugin/gradle/ArrowKotlinGradleSubplugin.kt

```diff
@@ -13,9 +13,9 @@ import org.jetbrains.kotlin.gradle.plugin.SubpluginOption
  */
 class ArrowKotlinGradleSubplugin : KotlinGradleSubplugin<AbstractCompile> {
   companion object {
-    private const val GRADLE_ARTIFACT_NAME = "gradle-plugin"
+    private const val META_ARTIFACT_NAME = "compiler-plugin"
```

Comment:
I checked this change when preparing #69. However, it seems that both `gradle-plugin` and `compiler-plugin` don't provide an additional feature for including the `compiler-plugin` with `-Xplugin`.